chumomega/echo_chamber
# add avg chamber status to chrome extension

## Description
server:
- add method to get the average of labels
- store in firebase when done
- update endpoint to return this status

frontend: 
- add chamber status component with reasoning

## Testing 
- Load extension
- go to youtube page
- click extension
- you should see chamber status show up with a button to find out why (why it is or why not)
- go to non youtube page
- you should only see hello
- see attached screenshot 
<img width="623" alt="Screenshot 2024-07-20 at 11 26 08 AM" src="https://github.com/user-attachments/assets/51bce1f3-478f-45ef-9a6d-7ca9739d77e8">
